packages/core/src/hooks/form/useForm.ts

```diff
@@ -554,6 +565,7 @@ export const useForm = <
                             }
                             reject();
                         },
+                        ...queryCacheUpdateMap,
```

Comment:
```suggestion
                        ...queryCacheUpdateMap,
```
We should pass this prop to first parameter of the mutation.